department-of-veterans-affairs/caseflow
# Queue | Colocated tweaks

Resolves #6855 

### Description
Please explain the changes you made here.

### Acceptance Criteria
Co-located user
- [x] Add title in header font to co-located queue = "Your Cases"
- [x] Add notes box when co-located user places hold
- [x] Change Pending tab to read Pending action
- [x] Co-located user can add another hold when a case is in the Pending Action tab
- [x] Prevent user from typing in hold select dropdown

Attorney user
- [x] Change copy from "Add Colocated task" to "Add admin action" 

### Testing Plan
1. Go to `/queue` as `BVALSPORER`
2. Place hold on task, write note. Go into case details view for that case, confirm note appears in CaseSnapshot (as Task Instructions)
3. Confirm Place Hold option appears in ColocatedActionsDropdown for cases in Pending Action tab
